doc/ipython-notebooks/converter/HessianLLE.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "#The scikit-example for this notebook\n",
+    "#is here - http://scikit-learn.org/stable/auto_examples/manifold/plot_lle_digits.html\n",
+    "\n",
+    "from scipy.io import loadmat, savemat\n",
+    "from numpy    import random\n",
+    "from os       import path\n",
+    "import matplotlib.pyplot as plt\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "\n",
+    "\n",
+    "mat  = loadmat('usps.mat')\n",
```

Comment:
you need to make sure this works when executed from its folde ron our servers. See the other notebooks for the relative path of the usps data